Automattic/wp-calypso
# Add/plans copy test

## Hypothesis
Variation `modified` will increase purchase rate by a minimum of 2%

## Variations
`original` 
![image](https://user-images.githubusercontent.com/6981253/27518147-bd6150ca-59a6-11e7-8b51-37b5990a0e84.png)

`modified`
This variation shows different copy depending on the options people select on the layout step in signup. It also includes a new mobile version:

**blog**
![screen shot 2017-06-25 at 12 56 50 pm](https://user-images.githubusercontent.com/6981253/27518148-c7fa69c2-59a6-11e7-9e70-94464f34c74a.png)

**website**
![screen shot 2017-06-25 at 12 57 00 pm](https://user-images.githubusercontent.com/6981253/27518151-c7fc4292-59a6-11e7-8497-cacb2ecb1a87.png)

**portfolio**
![screen shot 2017-06-25 at 12 57 10 pm](https://user-images.githubusercontent.com/6981253/27518150-c7fba788-59a6-11e7-97bb-cf5d7279246d.png)

**Mobile**
<img width="373" alt="screen shot 2017-06-25 at 12 57 37 pm" src="https://user-images.githubusercontent.com/6981253/27518149-c7fb423e-59a6-11e7-8826-82043742c7ad.png">

## Test References
**Test name** signupPlansCopyChanges
**Author** @fditrapani
**Domain** Signup
**Traffic Allocation control** 50% – variation: 50%

## Testing steps
* Start by creating a new account or creating a new site with an existing account by visiting `calypso.localhost:3000/start`.
* Set your cookie to see the variation:
`localStorage.setItem( 'ABTests', '{"signupPlansCopyChanges_20170623":"modified"}' );`
* Pick website/blog/portfolio from layout.
* Complete signup steps until you reach plans. 

Tested on: Chrome Mac, Firefox Mac, Opera Mac.
